src/platform/python/config.json.in

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Why JSON? I get the idea of stashing stuff in a format that Python can easily load out, but this seems...really quite overkill.